web/vtctld2/src/app/dashboard/shard.component.ts

```diff
@@ -184,35 +216,33 @@ export class ShardComponent implements OnInit, OnDestroy {
   }
 
   openEmergencyRepShardDialog() {
-    this.dialogSettings = new DialogSettings('Reparent', `Emergency Reparent Shard`, '',
-                                             'There was a problem initializing a shard master for {{shard_ref}}:');
+    this.dialogSettings = new DialogSettings('Reparent', `Emergency Reparent in Shard ${this.keyspaceName}/${this.shardName}`, '',
+                                             'There was a problem emergency reparenting shard {{shard_ref}}:');
```

Comment:
"Emergency reparenting" doesn't seem to be in English. Better to use "problem with emergency reparenting".